include/pmacc/math/complex/Bessel.hpp

```diff
@@ -0,0 +1,252 @@
+/* Copyright 2003-2017 Alexander Debus, C. Bond
```

Comment:
usually below is good enough in the `@file` description if derived, but in this case it seems to be close to the original author's version so why not.